content/en/docs/instrumentation/net/automatic/_index.md

```diff
@@ -0,0 +1,281 @@
+---
+title: Automatic Instrumentation
+description: Send traces and metrics from .NET applications and services.
+linkTitle: Automatic
+cSpell:ignore: coreutils HKLM iisreset myapp
+weight: 20
+---
+
+Use the OpenTelemetry .NET Automatic Instrumentation to send traces and metrics
+from .NET applications and services to observability back ends without having to
+modify their source code.
+
+To learn how to manually instrument your service or app code, see
+[Manual instrumentation](../manual).
+
+## Compatibility
+
+OpenTelemetry .NET Automatic Instrumentation should work with all officially
+supported operating systems and versions of
+[.NET](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core).
+
+The minimal supported version of
+[.NET Framework](https://dotnet.microsoft.com/download/dotnet-framework) is
+`4.6.2`.
+
+CI tests run against the following operating systems:
+
+- [Alpine](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docker/alpine.dockerfile)
+- [CentOS 7](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docker/centos-build.dockerfile)
+- [macOS Big Sur 11](https://github.com/actions/runner-images/blob/main/images/macos/macos-11-Readme.md)
+- [Microsoft Windows Server 2022](https://github.com/actions/runner-images/blob/main/images/win/Windows2022-Readme.md)
+- [Ubuntu 20.04 LTS](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md)
+
+{{% alert title="Note" color="note" %}} ARM architectures are not supported. See
```

Comment:
```suggestion
{{% alert title="Note" color="warning" %}} ARM architectures are not supported. See
```

The color `note` does not exist

Reply:
Oh. Do we have something yellowish or less dramatic than red? Or is that "warning"?

Reply:
warning is orange, info is blue